src/env-inl.h

```diff
@@ -113,6 +116,10 @@ inline AliasedFloat64Array& AsyncHooks::async_ids_stack() {
   return async_ids_stack_;
 }
 
+inline v8::Local<v8::Array> AsyncHooks::execution_async_resources() {
+  return execution_async_resources_.Get(env()->isolate());
```

Comment:
Tiny optimization tip: `execution_async_resources_` is a strong reference, for which we have an optimization that is kind of hacky but also faster than manually allocating a handle:

```suggestion
  return PersistentToLocal::Strong(execution_async_resources_);
```

Reply:
Neat. Didn't know about that. Thanks! :smile_cat: